### DISCUSSION TITLE
Can deepmd-kit starts training with existing VASP data set?

### DISCUSSION THREAD

**Discussion by Thesis-Appendix** [deepmodeling/deepmd-kit#304] [Q&A]
Hi, Dear developers:

Recently, I want to train a NN potential with the DeepMD-kit which is a powerful tool. In the Manual, the train data sets are extracted from MD trajectory. I have already calculated thousands of structures with VASP, So, can I train a NN potential with these data? If can, what show I do?

Hope for your reply!
Best

**Comment by jameswind** [comment]
Sure. You can use dpdata (
https://github.com/deepmodeling/dpdata) to convert the VASP data to the
format that DPMD recognizes, and use deepmd-kit to train (
https://github.com/deepmodeling/deepmd-kit#train-a-model).
